mitodl/hugo-course-publisher
# resolve course uid's

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/hugo-course-publisher/issues/136

#### What's this PR do?
Updates markdown with resolved course links.

#### How should this be manually tested?
Run the site, ensure that a course with links to other courses (such as the syllabus on 8.03SC) properly links to the indicated course.
